code/pluginbuild.xml

```diff
@@ -6229,6 +6229,13 @@ jar-all-plugins - Generate the plugin jar files
 				</patternset>
 			</fileset>
 		</jar>
+		<jar jarfile="${systemlstplugins.dir}/GameMode-CodeControlToken-STATSCORE.jar" manifest="${src.java.dir}/plugin/lsttokens/manifest.mf">
```

Comment:
Not auto generated, but happy to go there if you want to do that as a separate PR